NMS-15084: create basic content (#35)
OpenNMS/opennms-velocloud-plugin: docs/modules/reference/pages/config-properties.adoc

> = Configuration Properties

Author: Fixed.

:imagesdir: ../assets/images

> Some configuration properties allow to fine-tune the plugin's behaviour:

Reviewer: ```suggestion
Some configuration properties let you fine-tune the plugin's behavior:
```
Ugh. American spelling. :-)
